Missing generated file: `static/generated/plugins.json`

### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported


### Fastify version

latest

### Plugin version

Docusaurus version: 2.2.0

### Node.js version

18.13.0

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

20.04

### Description

when running `npm run build:webuild` , the error bellow is returned

```sh
[ERROR] Error: Missing generated file: static/generated/plugins.json. Run the "npm run build:website" once.
    at /home/edna/Desktop/bahati/Workspace/fastify/website-next/docusaurus.config.utils.js:27:13
    at Array.forEach (<anonymous>)
    at Object.checkGeneratedData (/home/edna/Desktop/bahati/Workspace/fastify/website-next/docusaurus.config.utils.js:24:18)
    at Object.<anonymous> (/home/edna/Desktop/bahati/Workspace/fastify/website-next/docusaurus.config.js:14:3)
    at Module._compile (node:internal/modules/cjs/loader:1218:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
    at Module.load (node:internal/modules/cjs/loader:1081:32)
    at Module._load (node:internal/modules/cjs/loader:922:12)
    at Module.require (node:internal/modules/cjs/loader:1105:19)
    at module.exports (/home/edna/Desktop/bahati/Workspace/fastify/website-next/node_modules/import-fresh/index.js:32:59)

```

seems not to complain about this on commenting out `'static/generated/plugins.json',` in https://github.com/fastify/website-next/blob/7bec1484ecf970aab3b9ba591869a48536e120ed/docusaurus.config.utils.js#L21

### Steps to Reproduce

run `npm run build:website`

### Expected Behavior

_No response_

--- a/scripts/build-website.sh
+++ b/scripts/build-website.sh
@@ -10,10 +10,10 @@ npm --prefix ./scripts install
 
 ####### Download Phase
 
-sh ./scripts/download-releases.sh --major 1
+./scripts/download-releases.sh --major 1
 # TODO download v2
-sh ./scripts/download-releases.sh --major 3
-sh ./scripts/download-releases.sh --minor 4
+./scripts/download-releases.sh --major 3
+./scripts/download-releases.sh --minor 4
 
 ####### Process Markdown Phase
 

--- a/scripts/download-releases.sh
+++ b/scripts/download-releases.sh
@@ -23,7 +23,10 @@ downloadRelease() {
   echo "Downloaded $repo > $1"
 
   mkdir -p "$baseDir/releases/$1/"
-  tar -xzf $outputFile -C "$baseDir/releases/$1/" --include "*/docs/*" --strip-components=1
+  mkdir -p "$baseDir/downloads/$1/"
+  tar -xzf $outputFile -C "$baseDir/downloads/$1/" --strip-components=1
+  cp -r "$baseDir/downloads/$1/docs" "$baseDir/releases/$1/docs"
+  rm -rf "$baseDir/downloads/$1/"
 }
 
 gh release list --repo $repo --limit 999 --exclude-drafts \

